CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.8.0rc1] - 2024-12-17
 
 ### Added
+- Scientific notation for frequency range warnings.
```

Comment:
This changelog item should be in Unreleased not in 2.8.0rc1. Also generally we tend to put new things at the *bottom* of the list (Added/Fixed/Changed), not at the top.